src/components/Posts/ComposePost.tsx

```diff
@@ -35,6 +35,17 @@ const ComposePost = ({ placeholder }: ComposePostProps) => {
   const [postTextContent, setPostTextContent] = useState("");
   const user = useAppSelector((state) => state.user);
   const dispatch = useAppDispatch();
+  const buttonStyle = {
+    postButton: {
+      "&.Mui-disabled": {
```

Comment:
Can we look at putting this in our theme so that all of our disabled buttons look the same? Same thing with the border radius. Might be a good idea for us to all hop on call go through this together sometime

Reply:
Sounds g lets bring this up next meeting

Reply:
Did this together